<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Elpida::NumberInputView</class>
 <widget class="QWidget" name="Elpida::NumberInputView">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="lblPropertyName">
     <property name="text">
      <string>Property Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="dsbNumber">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbUnitScale">
     <property name="insertPolicy">
      <enum>QComboBox::NoInsert</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbStandard"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
